python/ray/train/tests/test_lightgbm_predictor.py

```diff
@@ -97,9 +97,8 @@ def test_predict_feature_columns_pandas():
 
 
 def test_predict_no_preprocessor_no_training():
-    with tempfile.TemporaryDirectory() as tmpdir:
-        checkpoint = LightGBMCheckpoint.from_model(booster=model, path=tmpdir)
-        predictor = LightGBMPredictor.from_checkpoint(checkpoint)
+    checkpoint = LightGBMCheckpoint.from_model(booster=model)
```

Comment:
I think if we need to test checkpoint, creating tmpdir in unittest is better than surfacing it to class checkpoint implementation level